### PR TITLE
Bugfix: Apply app timer decrementing on app focus loss / regain just for Android

### LINKED ISSUE #71: Clicking back to window causes timer to drop sharply
# User Bug Report

For example, if I start the timer at 25:00, then navigate away at 24:00, then return at 22:00, then the time remaining drops to 21:00.

### PR DESCRIPTION
**Pull request by adrian-miasik**
Fixes #71 

---

For desktop platforms when the application is out of focus it can still continue to run in the background. This doesn't seem to be the case for Android apps. This is relevant since when the application is not in focus and the user tabs back into the android app, we want to decrement the timer to where it needs to be depending on when the app stopped running. (because again, Android apps don't run in the background - Desktop apps do). So when we implemented this into Unity Pomodoro, we introduced an unintended side effect for the desktop version. When users tab back / focus back into the application on desktop, the timer has been running in the background and is displaying correctly, but then on top of this, we also decrement the timer based on when the app lost focus and regained focus.

In conclusion, we are now decrementing the timer based on app focus loss just on Android, since Unity Pomodoro runs in the background for desktop apps and this added behaviour is unnecessary for desktop.

### CODE REVIEW

**Comment by adrian-miasik** [comment]
Fix confirmed for Desktop, testing Android next...

**Comment by adrian-miasik** [comment]
Fix confirmed for Android. App decrementation is now only applied on Android and ignored for desktop. Good to merge!
